Formula/biosig.rb

```diff
@@ -25,6 +25,8 @@ class Biosig < Formula
     sha256 "75df4a79b8d3d785942cbfd125ce45de49c3e7fa2cd19adb70caf8c4e30e13f0"
   end
 
+  patch :DATA
```

Comment:
Can you add a comment here explaining the patch? What problem does it solve/why is it needed? Will it be addressed upstream?

Is this Homebrew-specific or a general macOS compatibility issue?

Reply:
The installation with homebrew failed for two reasons:
1) the command line tools biosig_fhir and save2gdf did not build, because libbiosig was not available in the Library search path
2) the strip command on libbiosig.* failed 

The first problem seems to be specific to homebrew (./configure && make work fine, when compiling the same package from source outside of homebrew. )

The strip command seems to be a general macOSX compatibility issue, or it might not be needed on MacOSX. 

Whether or how to address the issue upstream, I do not know yet, I'll need to investigate in order to fully understand the problem, and how to best address these issue. Moreover, the checks from jenkins failed, and I do not understand why. The tests on my local test machine worked fine.

Reply:
> Moreover, the checks from jenkins failed, and I do not understand why.

Our CI nodes have network issues just now so are failing to clone from GitHub. I'll restart the job when things are operational again.

Reply:
biosig v2.0.1 addresses the issue with the strip command. As a side-effect also the first issue is gone.